recipes-nodes/rbuilder/rbuilder.inc

```diff
@@ -33,10 +34,11 @@ USERADD_PARAM:${PN} = "-r -s /bin/false -G eth rbuilder"
 do_install:append() {
     install -d ${D}${sysconfdir}/init.d
     install -m 0755 ${THISDIR}/init ${D}${sysconfdir}/init.d/rbuilder
-    
+
     # Install config files owned by root, readable by rbuilder
-    install -m 0744 ${THISDIR}/config ${D}${sysconfdir}/rbuilder.config
-    install -m 0744 ${THISDIR}/ofac.json ${D}${sysconfdir}/rbuilder.ofac.json
```

Comment:
why are we removing the ofac? are we getting it through some other ways now?

If it needs to be deleted, then also don't forget to delete the ofac.json file too and remove it from the SRC_URI above too

Reply:
We don't! It is still on the line 41